.github/workflows/ci.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           -e PYTEST_ADDOPTS=""
           -v ${GITHUB_WORKSPACE}:/src
           -v ${GITHUB_WORKSPACE}/.local:/.local
-          3liz/qgis-platform:3.16
+          3liz/qgis-platform:3.26
```

Comment:
Is this commit for QGIS Server 3.22 or 3.26 ?

Reply:
Minimum version is 3.22

Before merging would like to  parametrize the Qgis version with QGIS_VERSION this will enable the ci to have a test matrix